src/modules/network/hooks/useCommandsSchema.js

```diff
@@ -15,5 +15,16 @@ export const useCommandSchema = () => {
     );
   }, [data]);
 
-  return { moduleCommandSchemas, ...rest };
+  const messagesSchemas = useMemo(() => {
```

Comment:
The only thing these memos prevent is if `... rest` changes, ie, `isLoading, isFetching`. This is good to be aware of since it could cause unintended behaviour. 

if `data.commands` or actually `data.x` changes, this `useMemo` will still recalculate.

To fix that one would need to do:
```
  const messagesSchemas = React.useMemo(() => {
    // calcualte
  }, [data?.data?.messages]);
```
But even then, if we fetches and `data.data.messages` gets the same array, it would still recalculate. To prevent that one would need to use `JSON.stringify(` or `.length` since react only shallow compare (primitive values):

```
  const messagesSchemas = React.useMemo(() => {
    // calcualte
  }, [JSON.stringify(data.data.messages)]); // No advisable if not a heavy heavy calculation is performed inside
  OR
    }, [data.data.messages.length]); // Which is probably not what we want.
```

As you probably can tell, i am not a big fan of premature optimisation 😆